routes/api/contacts.js

```diff
@@ -1,25 +1,66 @@
-const express = require('express')
+const express = require("express");
+const {
+  listContacts,
+  getContactById,
+  addContact,
+  removeContact,
+  updateContact,
+} = require("../../models/contacts");
+const { contactValidator } = require("./../../utils/validators/validator");
 
-const router = express.Router()
+const router = express.Router();
 
-router.get('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+router.get("/", async (req, res, next) => {
+  const contacts = await listContacts();
+  console.log("GET /", contacts);
+  res.status(200).json(contacts);
+});
 
-router.get('/:contactId', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+router.get("/:contactId", async (req, res, next) => {
+  const { contactId } = req.params;
+  const contact = await getContactById(contactId);
+  if (contact) {
+    res.status(200).json(contact);
+  } else {
+    res.status(404).json({ message: "Not found" });
+  }
+});
 
-router.post('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+router.post("/", async (req, res, next) => {
+  const { error } = contactValidator(req.body);
+  if (error) return res.status(400).json({ message: error.details[0].message });
+  const contact = await addContact(req.body);
+  if (contact) {
+    res.status(200).json(contact);
+  } else {
+    res.status(400).json({ message: "missing required name field" });
```

Comment:
Widzę że w linii 31 obsługujesz już sytuację kiedy brakuje jakiś fields więc to nie jest potrzebne. A jeżeli w linii 31 nie sprawdzasz czy pole jest obence to trzeba by tam to dodać.